### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster @andrewconner

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/realtime/AppBoy.scala [hunk 1]
```diff
@@ -108,7 +108,7 @@ class AppBoy @Inject() (
           case UserPushNotificationCategory.ContactJoined => "us"
           case UserPushNotificationCategory.NewLibraryFollower => "nf"
           case UserPushNotificationCategory.NewOrganizationMember => "om"
-          case _ => throw new Exception(s"unsupported user push notification category ${upn.category.name}")
+          case _ => "us"
```
Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/realtime/AppBoy.scala]: This seems dangerous. Want to do the cool `AirbrakeStaticNotifier` thing?

Comment (camhashemi) [inline kifi-backend/modules/eliza/app/com/keepit/realtime/AppBoy.scala]: the catch-all being `"us"` it's actually the expected behavior (goes to the user's profile, we supply the message). without it, we were throwing exceptions on valid categories. I'll spell each category mapping out, but we should throw an exception if it fails. there doesn't seem to be an obvious best-effort resolution.

File: kifi-backend/modules/eliza/app/com/keepit/realtime/AppBoy.scala [hunk 2]
```diff
@@ -105,10 +105,14 @@ class AppBoy @Inject() (
       case upn: UserPushNotification =>
         val pushType = upn.category match {
           case UserPushNotificationCategory.UserConnectionRequest => "fr"
+          case UserPushNotificationCategory.UserConnectionAccepted => "us"
           case UserPushNotificationCategory.ContactJoined => "us"
           case UserPushNotificationCategory.NewLibraryFollower => "nf"
+          case UserPushNotificationCategory.NewLibraryCollaborator => "us"
+          case UserPushNotificationCategory.LibraryInviteAccepted => "us"
           case UserPushNotificationCategory.NewOrganizationMember => "om"
           case _ => throw new Exception(s"unsupported user push notification category ${upn.category.name}")
```
Comment (camhashemi) [inline kifi-backend/modules/eliza/app/com/keepit/realtime/AppBoy.scala:114]: @ryanpbrewster

Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/realtime/AppBoy.scala:114]: :+1:

**Pull request merged**